### ISSUE TITLE
Synthetic data generation

### ISSUE THREAD

**Issue by kamasyali** [d9n13lt4n/timegan-pytorch#4]
When I implement this with the stock data, the generated synthetic data is not so good. For example, opening price of a day should be very close to the closing price of the previous day. However, for some days I do not observe this pattern. 
Did you actually generate synthetic data for stock dataset? If yes, can you also share your generated data?

**Comment by ghost** [comment]
> ...opening price of a day should be very close to the closing price of the previous day. However, for some days I do not observe this pattern.

I think it would be normal that "some days" would have the poor results that you mentioned for a few reasons.
1. The hyperparameters were not optimized for this dataset. 
2. I don't think the evaluation metric (TSTR) would be sensitive to this issue. Therefore, there is a possibility that the model generated these "noisy" examples. 

> Did you actually generate synthetic data for stock dataset? If yes, can you also share your generated data?

Unfortunately, I did not save my results on the stock data, as I am using it for other purposes.  

You could try to generate samples using the original implementation [here](https://github.com/jsyoon0823/TimeGAN). Hope this helps.
